Dockerfile

```diff
@@ -7,7 +7,7 @@ ARG USER_GID
 
 RUN apt-get update
 RUN apt install sudo
-RUN useradd -ms /bin/bash $USER_NAME
+RUN useradd -ms /bin/bash $USER_NAME --no-log-init
```

Comment:
I had some issues without this. Maybe related to https://forums.docker.com/t/run-adduser-seems-to-hang-with-large-uid/27371